core/server/api/v2/db.js

```diff
@@ -46,9 +46,7 @@ module.exports = {
                 value: () => (exporter.fileName())
             }
         },
-        permissions: {
-            method: 'exportContent'
```

Comment:
Wasn't sure about how convention here works. From my understanding when using just `permissions: true` it's a shortcut for `{method: {apiConfig.methodName}}`, right?

Reply:
Yeah exactly, `permissions: true` will use the key for the current object in the apiDefinition. in this case they are the same :relaxed: